tools/packaging/static-build/ovmf/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2022 IBM
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ubuntu:20.04
+ENV DEBIAN_FRONTEND=noninteractive
+ENV INSTALL_IN_GOPATH=false
```

Comment:
As `yq` is not installed anymore, this bit should be dropped.

And once it's dropped, we're good to have this one merged.  Thanks @Alex-Carter01!